File: examples/twitter/search/node/searchForTweets.js
import * as twitter from "m3o/twitter";

// Search for tweets with a simple query
async function SearchForTweets() {
  let twitterService = new twitter.TwitterService(process.env.MICRO_API_TOKEN);
  let rsp = await twitterService.search({
    query: "cats",
  });
  console.log(rsp);
}

await SearchForTweets();

File: examples/go/twitter/search/searchForTweets.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/twitter"
)

// Search for tweets with a simple query
func SearchForTweets() {
	twitterService := twitter.NewTwitterService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := twitterService.Search(&twitter.SearchRequest{
		Query: "cats",
	})
	fmt.Println(rsp, err)
}
